seatunnel-connectors-v2/connector-prometheus/src/main/java/org/apache/seatunnel/connectors/seatunnel/prometheus/source/PrometheusSource.java

```diff
@@ -17,48 +17,30 @@
 
 package org.apache.seatunnel.connectors.seatunnel.prometheus.source;
 
-import org.apache.seatunnel.shade.com.typesafe.config.Config;
-
-import org.apache.seatunnel.api.common.PrepareFailException;
+import org.apache.seatunnel.api.configuration.ReadonlyConfig;
 import org.apache.seatunnel.api.source.Boundedness;
 import org.apache.seatunnel.api.table.type.SeaTunnelRow;
-import org.apache.seatunnel.common.config.CheckConfigUtil;
-import org.apache.seatunnel.common.config.CheckResult;
 import org.apache.seatunnel.common.constants.JobMode;
-import org.apache.seatunnel.common.constants.PluginType;
-import org.apache.seatunnel.common.exception.CommonErrorCode;
 import org.apache.seatunnel.connectors.seatunnel.common.source.AbstractSingleSplitReader;
 import org.apache.seatunnel.connectors.seatunnel.common.source.SingleSplitReaderContext;
 import org.apache.seatunnel.connectors.seatunnel.http.source.HttpSource;
-import org.apache.seatunnel.connectors.seatunnel.prometheus.Exception.PrometheusConnectorException;
-import org.apache.seatunnel.connectors.seatunnel.prometheus.config.PrometheusSourceConfig;
+import org.apache.seatunnel.connectors.seatunnel.prometheus.config.PrometheusQueryType;
+import org.apache.seatunnel.connectors.seatunnel.prometheus.config.PrometheusSourceOptions;
 import org.apache.seatunnel.connectors.seatunnel.prometheus.config.PrometheusSourceParameter;
 
 import lombok.extern.slf4j.Slf4j;
 
-import static org.apache.seatunnel.connectors.seatunnel.prometheus.config.PrometheusSourceConfig.INSTANT_QUERY;
-import static org.apache.seatunnel.connectors.seatunnel.prometheus.config.PrometheusSourceConfig.QUERY_TYPE;
-import static org.apache.seatunnel.connectors.seatunnel.prometheus.config.PrometheusSourceConfig.RANGE_QUERY;
-
 @Slf4j
 public class PrometheusSource extends HttpSource {
 
```

Comment:
I find the Prometheus connector is also base on Http Connector.
Should we move this to `connector-http` module?

Reply:
+1. We can do it with another PR. :)